app/views/categories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class=" b w-100 d-flex bg-primary text-white p-4 mb-2">
```

Comment:
- [ ] Please, consider that actually after signing up on your application, I forwarded to this error page. It seems something is wrong on the back end side. 
<img width="1116" alt="Screen Shot 2022-05-28 at 11 50 51" src="https://user-images.githubusercontent.com/49064106/170816497-e1f64e04-98db-481c-a512-2cc520897b6a.png">

Reply:
Hello @elmar8287 ,

My app works fine  without any issues. sometimes heroku automatically converts jpeg files to jpg which may cause the page fail to load.

<img width="1321" alt="Screenshot 2022-05-28 at 13 22 35" src="https://user-images.githubusercontent.com/15736819/170821226-906b5cc4-8632-461c-a23f-dbe942e27538.png">

Reply:
@billionsjoel if the app works properly, then just ask for next review 👍 Happy coding.